### ISSUE TITLE
"info -d" console output should be clearer

### ISSUE THREAD

**Issue by egypt** [rapid7/metasploit-framework#6964]
`info -d` tries to open a browser, which isn't useful on a headless server system. The output should be clearer on what it's doing.

## Steps to reproduce

1. `use exploit/windows/smb/psexec` (the particular module doesn't matter)
2. `info -d`

## Expected behavior

- Tell me that you're trying to open a browser on the file  
- Tell me where the file was written so i can open it myself.

E.g.:

```
[*] Generating documentation for psexec
[*] Opening a browser pointed at file:///tmp/psexec_whatever.html
```

## Current behavior

Prints 
```
[*] Please wait, generating documentation for psexec
```
On headless systems it looks like a silent failure when nothing else happens because there is no browser.

**Comment by h00die** [comment]
adding a print_status is easy... however, getting that function imported (when unfamiliar w/ ruby...)
I have tried several combos to get print_status imported including (many combinations of):

Inside documentation_generator.rb:
require 'msf/core/module/ui/message'

and inside Module DocumentationGenerator:
include Msf::Module::UI::Message
print_status()
Msf::Module::UI::Message::print_status()

but every print_status causes a stack_trace where it can't find `print_status`.

Throw me a bone :)

**Comment by wchen-r7** [comment]
Indeed a little tricky because the API that knows where the file path is in Msf::Util::DocumentGenerator#spawn_module_document, and in there I guess you can't call print_status. So it's not as straight forward as putting print_status in there and that's it. One way around is we change:

```ruby
def self.spawn_module_document(mod)
```

To:

```ruby
def self.spawn_module_document(mod, out_file)
```

And then when you call from core.rb, you can do:

```ruby
f = Rex::Quickfile.new(["#{active_module.shortname}_doc", '.html'])
begin
  print_status("Opening #{f.path} for #{active_module.shortname} from a browser...")
  Msf::Util::DocumentGenerator.spawn_module_document(active_module, f)
ensure
  f.close if f
end
```

Something like that.... looks a little weird though.

**Comment by wchen-r7** [comment]
Resolved in #6977
